packages/components/src/tab-panel/index.tsx

```diff
@@ -210,6 +210,9 @@ const UnforwardedTabPanel = (
 								}
 							) }
 							disabled={ tab.disabled }
+							aria-controls={ `${ prependInstanceId(
+								tab.name
+							) }-view` }
```

Comment:
The current `TabPanel` component adds `aria-controls` attributes to all of the `Tabs`, even though with lazy loading only the currently active tab was actually rendered. Ariakit was only adding this attribute for the current tab, because technically it's the only one that exists. I've manually added this in for consistency to match existing snapshots as closely as possible, but we can easily remove it and update the snapshots to expect the new behavior instead.